backfill_corrections/Makefile

```diff
@@ -33,19 +53,41 @@ run-local:
 	[ "$$?" -eq 1 ]
 
 run:
-	# Assumes receiving is output dir
 	docker run --rm --pull=always \
-	    -v "${PWD}"/logs:/backfill_corrections/logs \
-	    -v "${PWD}"/receiving:/backfill_corrections/receiving \
-	    -v "${PWD}"/input:/backfill_corrections/input \
-	    -v "${PWD}"/cache:/backfill_corrections/cache \
+	    -v "${PWD}/${LOG_DIR}:/backfill_corrections/${LOG_DIR}" \
+	    -v "`realpath $(USR_EXPORT_DIR)`:/backfill_corrections/${EXPORT_DIR}" \
```

Comment:
what's `realpath`?

Reply:
It turns a relative path into an absolute path, which Docker volume mounting requires. I was previously combining the full path of the current working dir with the various params-provided directories, but realized there's no requirement that those directories actually exist in the current working dir or that they're not already abs paths. Thought this would be more flexible.

We may need to install `realpath`.